client/src/pages/Chat.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo, useState } from 'react';
+import { Message, MessageProps } from '../components/Message';
+import { ChatTopBar } from '../components/ChatTopBar';
+import { ChatInput } from '../components/ChatInput';
+import '../styles/Chat.scss';
+
+export const Chat = () => {
+  const [msg, setMsg] = useState<MessageProps[]>([
+    { isSender: false, content: 'הודעה ראשונה', id: 1 },
+    { isSender: false, content: 'הודעה שנייה', id: 2 },
+    { isSender: true, content: 'תגובה ראשונה אמור להיות בסדר ', id: 3 }
+  ]);
```

Comment:
@YaelEdri - You can create a mock file, with this data, it will be cleaner.

Reply:
OK